### PR TITLE
Feature/581/add storybook test radio button input

### PR DESCRIPTION
**Pull request by ArturBekhDEV**
Write unit test and story book to RadioButtonInputs component.

### CODE REVIEW

File: tests/unit/components/radio-button-inputs/RadioButtonInputs.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  title: 'Rating'
+}
+
+describe('RadioButtonInputs test', () => {
+  beforeEach(() => {
+    render(<RadioButtonInputs { ...mockedProps } />)
+  })
+
+  it('Should render radio buttons title correctly'),
+  () => {
```
Comment (Yurenko) [inline tests/unit/components/radio-button-inputs/RadioButtonInputs.spec.jsx:22]: The syntax is wrong.
Must be: it('Should render radio buttons title correctly',  () => {

File: tests/unit/components/radio-button-inputs/RadioButtonInputs.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    const radioButtonTitle = screen.getByLabelText('First Radio Title')
+
+    expect(radioButtonTitle).toBeInDocument()
+  }
+  it('Should call onChange function with the selected value'),
+  () => {
```
Comment (Yurenko) [inline tests/unit/components/radio-button-inputs/RadioButtonInputs.spec.jsx:28]: Here, the same

File: tests/unit/components/radio-button-inputs/RadioButtonInputs.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    const radioButton = screen.getByLabelText('Third Radio Title')
+
+    fireEvent.click(radioButton)
+
+    expect(mockedProps.onChange).toHaveBeenCalled(1)
+    expect(mockedProps.onChange).toHaveBeenCalledWith('Third Radio Title')
+  }
+  it('Should render title if it provided'),
+  () => {
```
Comment (Yurenko) [inline tests/unit/components/radio-button-inputs/RadioButtonInputs.spec.jsx:37]: And here

File: src/stories/RadioButtonInputs.stories.jsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import RadioButtonInputs from '~/components/radio-button-inputs/RadioButtonInputs'
+import { useState } from 'react'
```
Comment (Yurenko) [inline src/stories/RadioButtonInputs.stories.jsx:2]: Establish the correct order of imports

File: tests/unit/components/radio-button-inputs/RadioButtonInputs.spec.jsx (hunk 1 above)
Comment (ArturBekhDEV) [reply tests/unit/components/radio-button-inputs/RadioButtonInputs.spec.jsx]: fixed

File: tests/unit/components/radio-button-inputs/RadioButtonInputs.spec.jsx (hunk 3 above)
Comment (ArturBekhDEV) [reply tests/unit/components/radio-button-inputs/RadioButtonInputs.spec.jsx]: fixed and updated tests

File: src/stories/RadioButtonInputs.stories.jsx (hunk 4 above)
Comment (ArturBekhDEV) [reply src/stories/RadioButtonInputs.stories.jsx]: fixed

**Review by MickhaelSh — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Review by Yurenko — APPROVED** [review]

**Review by Tolik170 — APPROVED** [review]

**Pull request merged**